androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/triage/suicideassessment/SuicideAssessmentScreen.kt

```diff
@@ -15,17 +17,17 @@ import com.EENX15_22_17.digital_journal.android.ui.theme.Colors
 
 @Composable
 fun SuicideAssessmentScreen(
-    onBackClicked: () -> Unit = {},
-    onMenuClicked: () -> Unit = {},
+    onBackClicked: () -> Unit,
+    visitId: String
 ) {
     DetailPageWrapper(
-        title = "SUICIDBEDÖMNING", // TODO: Use `JournalScreen.ACTIONS_AND_ORDINATIONS.title` when libs are available,
+        title = stringResource(id = R.string.suicideAssessment),
         titleColor = Colors.treatmentPrimary,
         onBackClicked = onBackClicked,
-        onMenuClicked = onMenuClicked
+        onMenuClicked = {}
     ) {
         Box(
-            modifier = Modifier.fillMaxSize().background(Color.White)
+            modifier = Modifier.fillMaxSize()
```

Comment:
Single statement-modifier could be on same line I think. But then again, I don't know if this box is needed. Does it really have any effect when it contains a single child (a `Column` in this case)